hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java

```diff
@@ -123,4 +125,76 @@ public static Method getDeclaredMethod(Class<?> type, String methodName, Class<?
         return Optional.absent();
     }
 
+    public static String getMethodInfo(Method m) {
```

Comment:
May it make sense to add some information about the declaring class of the method?